docs/contributing/templates/c2c-source-and-app.md

```diff
@@ -0,0 +1,170 @@
+sidebar_label: App Template #example: Druva
+tags:
+  - cloud-to-cloud
+  - c2c-source #example: Druva-source
+  - app-doc #example: Druva
+description: Description goes here. #example: Learn how to configure, install, and view the Druva dashboards in your Sumo Logic environment.
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<img src={useBaseUrl('path-to-your-icon.png')} alt="Thumbnail icon" width="45"/>
+
+\Introduction\
+
+The {{source name}} collects {{data/event types}} from {{source of origin}}. 
+
+Example: The Microsoft Graph Security API Source provides a secure endpoint to consume alerts from the Microsoft Graph Security API endpoint. It securely stores the required authentication, scheduling, and state tracking information. One threat event is reported for each affected device.
+
+The Sumo Logic app for {{app name}} enables you to track {{What the app does}}.
+
+Example: The Sumo Logic app for Asana enables you to track user sign in activities, detect data access and export, and identify changes to security settings and user roles.
+
+{{Key features of the app}}
+
+\Depending on the availability in the Fed, add the below note.\
+
+:::note
+This source is **not** yet available in the [Fed deployment](https://help.sumologic.com/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security).
+:::
+
+## Data collected
+
+\Add all the data sources and respective pollig iterval information\
+
+| Polling Interval | Data |
+| :--- | :--- |
+| {{Polling time in minutes}} | {{Data sources}} |
+
+Example:
+
+| Polling Interval | Data |
+| --- | --- |
+| 5 min |  [Team Events](https://www.dropbox.com/developers/documentation/http/teams#team_log-get_events)
+
+### Sample log message
+
+```json title="example"
+Enter the log message in json format with the title.
+```
+
+### Sample Query
+
+```sql title="example"
+Enter the log message in sql format with the title.
+```
+
+## Setup
+
+To set up Cloud to Cloud Integration /source name/ Source for the /app name/ app, follow the instructions provided. These instructions will guide you through the process of creating a source using the /source name/ Source category, which you will need to use when installing the app. By following these steps, you can ensure that your /app name/ app is properly integrated and configured to collect and analyze your /app name/ data.
+
+### Vendor configuration
+
+#### Prerequisites
+
+\NOTE: This section doesn't apply to all sources; use only where needed\
+
+Example: You'll need a Dropbox App Key, App Secret, and Access Code to provide to Sumo Logic. To generate these credentials, ....
+
+\Insert steps to configure the Source in the Vendor UI\
+
+Example: https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dropbox-source/#vendor-configuration
```

Comment:
There is no #vendor-configuration section in that article. Did you want to choose another article that has a "Vendor configuration" section?

Reply:
Thanks @jpipkin1, I have updated the URL.